tests/artifact/contracts/duplicate_ic_contract_2.py

```
@@ -0,0 +1,22 @@
+# { "Depends": "py-genlayer:test" }
+
+from genlayer import *
+
+
+# contract class
+class DuplicateContract(gl.Contract):
```

Ruff (0.11.9)

`gl` may be undefined, or defined from star imports

(F405)